forge-gui/res/cardsfolder/upcoming/sokrates_athenian_teacher.txt

```diff
@@ -0,0 +1,12 @@
+Name:Sokrates, Athenian Teacher
+ManaCost:1 W U
+Types:Legendary Creature Human Advisor
+PT:0/4
+K:Defender
+S:Mode$ Continuous | Affected$ Card.Self+untapped | AddKeyword$ Hexproof | Description$ CARDNAME has hexproof as long as it's untapped.
+A:AB$ Animate | Cost$ T | ValidTgts$ Creature | PrecostDesc$ Sokratic Dialogue | Replacements$ RPrevent | SpellDescription$ Until end of turn, target creature gains "If this creature would deal combat damage to a player, prevent that damage. This creature's controller and that player each draw half that many cards, rounded down."
+SVar:RPrevent:Event$ DamageDone | ValidTarget$ Player | Prevent$ True | IsCombat$ True | ValidSource$ Card.Self | ReplaceWith$ Draw | Description$ If this creature would deal combat damage to a player, prevent that damage. This creature's controller and that player each draw half that many cards, rounded down.
+SVar:Draw:DB$ Draw | Defined$ ReplacedSourceController & CardController | NumCards$ X 
```

Comment:
isn't that the same player twice?

Reply:
Yeah, any ideas about the correct defined for "that player" ?

Reply:
Why not just "ReplacedTarget"?

Reply:
Thanks working now ! 
![image](https://github.com/Card-Forge/forge/assets/67333662/429a4d5c-36f0-4645-8029-ba20ae197d7d)